contracts/SpokePool.sol

```diff
@@ -1193,6 +1193,13 @@ abstract contract SpokePool is
             "invalid repayment chain"
         );
 
+        // If this is a partial fill, require that the fill has no message. This prevents situations in which
+        // a message is sent multiple times for multiple partial fills.
+        require(
+            relayExecution.relay.amount == fillAmountPreFees || relayExecution.updatedMessage.length == 0,
+            "invalid partial fill message"
```

Comment:
```suggestion
            "invalid partial fill (message)"
```

Reply:
you think this improves the error message a bit? I'm not convinced but curious what you were thinking?

Reply:
> you think this improves the error message a bit? I'm not convinced but curious what you were thinking?

Yeah..even though I know exactly what the issue is, I have trouble parsing "invalid partial fill message" in my head. How about this?

```suggestion
            "invalid message on partial fill"
```